lib/mongoid/history/trackable.rb

```diff
@@ -52,11 +52,20 @@ def dynamic_enabled?
           Mongoid::Compatibility::Version.mongoid3? || (self < Mongoid::Attributes::Dynamic).present?
         end
 
-        def disable_tracking(&_block)
-          Mongoid::History.store[track_history_flag] = false
-          yield
+        def disable_tracking(&block)
+          with_tracking(false, &block)
+        end
+
+        def enable_tracking(&block)
+          with_tracking(true, &block)
+        end
+
+        def with_tracking(state)
```

Comment:
Similar to the class methods, these should be called `with_tracking` and `without_tracking`. I would alias `disable_tracking` to `without_tracking` and never expose a method that has a boolean parameter. It's OK to duplicate code a little bit too IMO for something so trivial.